cmd/destroy/destroy.go

```diff
@@ -178,12 +188,23 @@ func Destroy(ctx context.Context) *cobra.Command {
 			}
 			os.Setenv("KUBECONFIG", kubeconfigPath)
 			defer os.Remove(kubeconfigPath)
-			err = c.runDestroy(ctx, options)
-			analytics.TrackDestroy(err == nil)
-			if err == nil {
-				oktetoLog.Success("Development environment '%s' successfully destroyed", options.Name)
-			}
 
+			// when option --all the cmd will destroy everything at the namespace and return
+			if options.DestroyAll {
+				if !okteto.Context().IsOkteto {
+					return errors.New("option `--all` is not available for non-Okteto clusters. Learn more: https://www.okteto.com/docs/self-hosted/")
```

Comment:
could we check here the copy of this message if is the correct one? 

cc: @ifbyol @pchico83 @rberrelleza @rlamana

Reply:
@teresaromero instead of failing, could we ignore the logic to destroy the pipelines in this case?

Reply:
@pchico83 you mean at the destroy-all job that is triggered?